drivers/gles3/shaders/scene.glsl

```diff
@@ -1614,16 +1614,12 @@ void main() {
 	float shadow2 = sample_shadow(directional_shadow_atlas, directional_shadows[directional_shadow_index].shadow_atlas_pixel_size, shadow_coord2);
 	float directional_shadow = 1.0;
 
-	if (depth_z < light_split_offsets.y) {
```

Comment:
Did you mean to delete this? If so you need to fix the rest of the code

Reply:
didn't mean to delete that